doc/examples.rst

```diff
@@ -387,3 +387,126 @@ Flag without argument:
    http://127.0.0.1:1024/tarantool
    {"res":"Tarantool CLI version 0.1.0, darwin/amd64. commit: bf83f33\n
     • Tarantool executable found: '/usr/local/bin/tarantool'\n"}
+
+Transition from tarantoolctl to tt
+----------------------------------
+Suppose we have a set of Tarantool instances managed by ``tarantoolctl`` utility. In order for the tt
```

Comment:
I think we should also describe the case where the user does not have a `.tarantoolctl' file but uses `tarantoolctl' to run tarantool instances on the system.

Reply:
Added a section for system-wide config.